### ISSUE TITLE
Window opacity causes window to not be shown at all

### ISSUE THREAD

**Issue by EaranMaleasi** [BornToBeRoot/NETworkManager#178]
**Describe your issue or feature request**:
Enabling window opacity causes NETworkManager to not be shown at all after restart. The opacity level is irrellevant. Disabling opacity in the settings file directly and restart restores the window.

**NETworkManager version**:
1.9.0.0

**Windows version**:
Windows 10 Pro 1809

**.NET-Framework version**:
.NET 4.7

**Screenshots**
N/A

Here my current settings file:
[Settings.txt](https://github.com/BornToBeRoot/NETworkManager/files/2874511/Settings.txt)

**Comment by BornToBeRoot** [comment]
Strange... i can't repoduce this issue.

What graphic card are you using?

Edit: Does the application crash? (EventLog)
Edit2: The settings file works without any issues on my system (nvidia).

**Comment by EaranMaleasi** [comment]
I have an Nvidia Geforce GT 710, driver Version 388.13. I updated it to the latest one, 419.17, which didn't help.   

What did help, was hovering the mouse over the icon on the taskbar so that the small preview appears and all other apps go invisible with only their borders highlightened so that you would see where the app is located beneath all of the bazillion open apps. Well it didn't show up at all and the preview just showed that placeholder icon for an application. Right-clicking on the preview gave me restore and close, everything else was grayed out. Clicking on restore made the window appear. 

So it works now. I wonder what caused this Issue. Maybe it has to do something with the options around the tray icon as I played around with them at the same time as I changed the opacity.
